src/composables/useTokenFiatMath.ts

```diff
@@ -0,0 +1,68 @@
+import { computed, Ref } from 'vue';
+import { bnum } from '@/lib/utils';
+import useNumbers, { fNum } from '@/composables/useNumbers';
+import useUserSettings from '@/composables/useUserSettings';
+
+export type InvestMathResponse = {
+  fullAmounts: Ref<string[]>;
+  fiatTotal: Ref<string>;
+  fiatTotalLabel: Ref<string>;
+};
+
+export default function useTokenFiatMath(
```

Comment:
I'm not sure about this naming. This composable is very specific to an array of token amounts but the naming doesn't suggest that. Maybe `useTokenAmountsMath` would be more appropriate?

Reply:
I got rid of this whole composable. It wasn't needed as we already have the USD summary from the summary page.